src/rdma.c

```diff
@@ -0,0 +1,1931 @@
+/* ==========================================================================
+ * rdma.c - support RDMA protocol for transport layer.
+ * --------------------------------------------------------------------------
+ * Copyright (C) 2021-2024  zhenwei pi <pizhenwei@bytedance.com>
+ *
+ * This work is licensed under License 1 of the COPYING file in
+ * the top-level directory.
+ * ==========================================================================
+ */
+
+#define VALKEYMODULE_CORE_MODULE
+#include "server.h"
+
+#ifdef USE_RDMA
+#ifdef __linux__    /* currently RDMA is only supported on Linux */
```

Comment:
It's better that the build fails when trying to build RDMA without linux?

```suggestion
#ifndef __linux__
#error Currently RDMA is only supported on Linux
#endif
```

Reply:
Sure,  the build check is at the bottom of the file.